src/window.hpp

```diff
@@ -47,6 +47,8 @@ namespace wr
 		std::int32_t GetHeight() const;
 		/*! Returns the native window handle (HWND)*/
 		HWND GetWindowHandle() const;
+		/*! Checks whether the window is fullscreen */
+		bool IsFullscreen() const;
```

Comment:
Should be removed. This is quite a poor way to do this. No good alternative at the moment so I'll leave this as is.